app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
```

Comment:
I see this as redundant. We should always have the current plan in our store, which contains any treatment that the user has access to. Should we just have the 'treatmentId' on the store and do a lookup in any connect component containing a treatment w/o the plan?